src/main/java/com/epam/team1/emotional/help/controller/ContactsController.java

```diff
@@ -16,13 +18,22 @@
 @CrossOrigin
 public class ContactsController {
     private final MailService mailService;
+    private final QuizService quizService;
+
 
     @GetMapping("/callRequest")
     public void callRequest(@Valid @RequestBody CallRequestDto dto){
+        log.info("Call of /contacts/callRequest endpoint.(Get method). dto is {}", dto);
```

Comment:
Try to describe what happens, without mentioning the HTTP method and the endpoint.